index.vue

```diff
@@ -113,6 +116,12 @@ export default
 			@fitsAssetsContainerClasses
 			@loadsAssetsContainerClasses
 		].filter (val) -> !!val
+	
+	methods:
+		togglePause: (e) -> 
+			if e.target.paused 
+			then e.target.play() 
+			else e.target.pause()
```

Comment:
@brokenhd can you move this method into https://github.com/BKWLD/vue-visual/blob/master/concerns/supports-videos.coffee since it's fundamentally video related? 

Also, how about `togglePlayback()` as the method name?

Lastly, you can use the existing `@play()` and `@pause()` methods for this, which would be good in case we add additional behaviors onto those methods.  Like how `@play()` also starts loading.